% update chapters
% update NSF grant
%________________________________________________________________________________________
% @brief    LaTeX2e Resume for C. Titus Brown
% @author   Titus Brown
% @date     January 2008
% @info     Based on Latex Resume Template by Chris Paciorek 
%           http://www.biostat.harvard.edu/~paciorek/


%________________________________________________________________________________________
\documentclass[margin,line]{resume}

\usepackage{hyperref}
\usepackage{color}
\hypersetup{urlcolor=blue}

\begin{document}
\name{\Large C. Titus Brown}
\pagenumbering{arabic}
\pagestyle{plain}


\begin{resume}


    %____________________________________________________________________________________

    {\small (As of September 2016.)}

    % Education
    \section{\mysidestyle Education}

    {\bf Reed College}, Portland, OR; Mathematics; B.A., 1997

    \vspace{2mm}

    {\bf California Institute of Technology}, Davidson Lab (graduate student);
\\
Developmental Biology; PhD., 2007

    \vspace{2mm}

    {\bf California Institute of Technology}, Bronner-Fraser Lab (postdoc);\\
 Developmental Biology and Bioinformatics; 2007-2008

    \vspace{2mm}

    \section{\mysidestyle Appointments}

    {\bf Assistant Professor}, 
Microbiology \& Molecular Genetics / Computer Science and Engineering\\
Michigan State University, 2008-2014.

    {\bf Associate Professor}, 
Population Health and Reproduction, School of Veterinary Medicine \\
University of California Davis, 2015-present.

    %____________________________________________________________________________________
    % Honours and Awards
    \section{\mysidestyle Honours and\\Awards} 

Burroughs-Wellcome Fund Computational Biology Fellowship (1999-2004).\\
Withrow Award for Teaching Excellence in Computer Science at MSU (2008-2009). \\
Woods Hole Marine Biological Laboratory Summer Fellow (2013).\\
Michigan State University / College of Natural Science Teacher-Scholar Award (2013).\\
{\bf Moore Foundation Data Driven Discovery Investigator (2014-2019).}


\section{\mysidestyle Grants}


15. Department of Homeland Security, \$248,000 (100\%); 12 months, award pending. ``High Confidence Metagenomics Analysis of Complex Samples.''

14. Sloan Foundation, \$20,000 (100\%); 7/1/16-12/31/16. ``A Workshop on Dockerized Notebook Computing.''

13. Moore Foundation, \$1,500,000 (100\%); 12/1/14-12/1/19.  ``Software to support data driven inquiry in biology.''

12. USDA, \$129,889 (100\%); 10/1/13-9/30/16.  Collaborative proposal with WUSTL. ``Improvement of the Chicken Reference Genome.''

11. NSF, \$273,419 (20\%); 9/2013-9/2016. co-PI, ``Supporting Efficient Discrete Box Queries for Sequence Analysis on Large Scale Genome Databases.''

10. NIH R25, \$162,000 (100\%); 7/2013-7/2016.  PI, ``Analyzing Next-Generation Sequencing Data.'' Renewal.

9. NIH R01, \$699,231 (100\%). 5/2013-5/2016. PI, ``BIGDATA: Low-Memory Streaming Prefilters for Biological Sequencing Data.''

8. USDA, \$2,989,032 (5.8\%). 1/2013-1/2017.  co-PI, ``The Genetics of Johne's Disease.''

7. NSF OCI Supplement to BEACON STC, \$200k (100\%); 1/1/2013-12/31/2013.  PI, ``Materials and Workshops for Cyberinfrastructure Education in Biology.''

6. USDA (Grad Fellowships), \$238,000 (15\%).  1/2012-11/2017.  co-PI, ``Integrated genomics training program.''

\vspace{2cm}
{\centerline {Page 1/7}}

\newpage

5. DOE, \$659,587 (5\%).  9/2011-8/2012.  co-PI, ``Ribosomal Database Project.''

4. NIH (R25 education), \$104,000 (100\%); 7/2011-6/2014.  PI, ``Analyzing Next-Generation Sequencing Data.''

3. USDA, \$99,000; 2/2009-12/2011 (100\%).  PI, ``Positional Candidate Genes for Resistance to Marek's
Disease by Screening for Marek's Disease Virus Meq-regulated Genes.''

2. NSF, \$50,000; 9/2009-8/2011 (100\%).  PI, ``RV1: MSB: Collaborative: Symbiont Separation and Investigation of the Novel Heterotrophic Osedax Symbiosis using Comparative Genomics.''

1. USDA, \$690,000; 12/2009-11/2013 (100\%).  PI, ``Easily accessible Web-based tools for analyzing next-generation sequencing data from agricultural animals.''

    %____________________________________________________________________________________
    % Publications

    \section{\mysidestyle Submitted Manuscripts}

Note: {\color{red} ${\bf \triangleright}$} highlights the most important
papers, in my opinion (personal contribution/impact).

{\color{red} ${\bf \triangleright}$}
{\em A Reference-Free Algorithm for Computational Normalization of Shotgun Sequencing Data.} {\bf Brown CT}, Howe AC, Zhang Q, Pyrkosz AB, Brom TH. preprint arXiv:1203.4802. (12 cit; Google Scholar incomplete.)

    \section{\mysidestyle Peer Reviewed Publications}

{\em Full publication list at: http://scholar.google.com/citations?user=O4rYanMAAAAJ\\
  March 2016: 5090 citations total; h-index of 29, i10-index of 38.}

{\em Transcriptome of the Caribbean stony coral Porites astreoides from three developmental stages.} Mansour TC, Rosenthal JJC, Brown CT, Roberson LM.
GigaScience 5 (1), 33, August 2016.

{\em How open science helps researchers succeed.}
McKiernan EC et al., eLife 5, e16800, July 2016.

{\em Haplotype-phased synthetic long reads from short-read sequencing.}
JA Stapleton, et al.,
PLOS One 11 (1), e0147229, 2016.

{\em Microbial community analysis with ribosomal gene fragments from shotgun metagenomes.} Guo J, Cole JC, Zhang Q, Brown CT, Tiedje JM. Applied and environmental microbiology. Jan 2016.

{\em Haplotype-phased synthetic long reads from short-read sequencing.} Stapleton JA, Kim J, Hamilton JP, Wu M, Irber LC, Maddamsetti R, Briney B, Newton L, Burton DR, Brown CT, Chan C, Buell CR, Whitehead TA. PLOS One. Jan 2016.

{\em Hsp90 and hepatobiliary transformation during sea lamprey metamorphosis.}
Chung-Davidson YW, Yeh CY, Bussy U, Li K, Davidson PJ, Nanlohy KG, Brown CT, Whyard S, Li W. BMC Developmental Biology. Dec 2015.

{\em Differentially-Expressed Pseudogenes in HIV-1 Infection.} Gupta A, Brown CT, Zheng YH, Adami C. Viruses. Sep 2015.

{\em The khmer software package: enabling efficient nucleotide sequence analysis.} Crusoe MR et al., Brown CT. F1000Research. Sep 2015.

{\em Xander: employing a novel method for efficient gene-targeted metagenomic assembly.} Wang Q, Fish JA, Gilman M, Sun Y, Brown CT, Tiedje JM, Cole JR. Microbiome. August 2015.

{\em Transcriptome variation in response to Marekâ€™s disease virus acute infection.} Preeyanon L, Brown CT, Cheng H. Cytogenetics. July 2015.

{\em Phylogeny and phylogeography of functional genes shared among seven terrestrial subsurface metagenomes reveal N-cycling and microbial evolutionary relationships.} Lau MCY et al., Onstott TC. Frontiers in Microbiology. Oct 2014.

\vspace{1cm}
{\centerline {Page 2/7}}

\newpage

{\em Divergent mechanisms regulate conserved cardiopharyngeal development and gene expression in distantly related ascidians.} Stolfi A, Lowe EK, Racioppi C, Ristoratore F, Brown CT, Swalla BJ, Christiaen L. eLife. Sep 2014.

{\em These are not the k-mers you are looking for: efficient online
  k-mer counting using a probabilistic data structure.} Zhang Q, Pell
J, Canino-Koning R, Howe AC, {\bf Brown CT}.  PLoS One. July 2014.

{\color{red} ${\bf \triangleright}$}
{\em Tackling soil diversity with the assembly of large, complex metagenomes.}
Howe AC, Jansson J, Malfatti SA, Tringe SG, Tiedje JM, {\bf Brown CT}. Accepted at PNAS, 2/2014.  preprint arXiv:1212.2832. (1 cit.)

{\em The Ribosomal Database Project: Data and Tools for High Throughput rRNA Analysis.} Cole JR, Wang Q, Fish J, Chai B, McGarrell D, Sun Y, Brown CT, Porras-Alfaro A, Kuske C, Tiedje JM.  Accepted, Nucleic Acid Res, Nov 2013.

{\em Genomic versatility and functional variation between two dominant
heterotrophic symbionts of deep-sea {\rm Osedax} worms.} Goffredi S, Yi H, Zhang Q, Klann J, Struve I, Vrijenhoek RC, {\bf Brown CT}. Accepted, ISME Journal, October 2013.

{\em Best practices for scientific computing.} Wilson GV et al. preprint arXiv:1210.0530. Accepted PLoS Biology, October 2013. (17 cit.)

{\em FunGene: the Functional Gene Pipeline and Repository.} Fish JA, Chai B, Wang Q, Sun Y, {\bf Brown C}, Tiedje JM and Cole JR (2013).  Front. Microbiol. 4:291. doi: 10.3389/fmicb.2013.00291

{\em The genome and developmental transcriptome of the strongylid
  nematode Haemonchus contortus.}  Schwarz EM, Korhonen PK, Campbell
BE, Young ND, Jex AR, Jabbar A, Hall RS, Mondal A, Howe AC, Pell J,
Hofmann A, Boag PR, Zhu XQ, Gregory TR, Loukas A, Williams BA,
Antoshechkin I, {\bf Brown CT}, Sternberg PW, Gasser RB.  Genome
Biol. 2013 Aug 28;14(8):R89.

{\em The sea lamprey has a primordial accessory olfactory system.}
Chang S, Chung-Davidson YW, Libants SV, Nanlohy KG, Kiupel M, {\bf
  Brown C.T.}, Li W. BMC Evol Biol. 2013 Aug 17;13(1):172. doi:
10.1186/1471-2148-13-172.

{\em Integrated Analyses of Genome-Wide DNA Occupancy and Expression
  Profiling Identify Key Genes and Pathways Involved in Cellular
  Transformation by a Marek's Disease Virus Oncoprotein, Meq.}
Subramaniam S, Johnston J, Preeyanon L, {\bf Brown CT}, Kung HJ, Cheng
HH. J Virol. 2013 Aug;87(16):9016-29. doi: 10.1128/JVI.01163-13.

{\em A thermogenic secondary sexual character in male sea lamprey.}
Chung-Davidson, Y.-W., Priess, M.C., Yeh, C.-Y., Brant, C.O., Johnson,
N.S., Li, K., Nanlohy, K.G., Bryan, M.B., {\bf Brown, C.T.}, Choi, J.,
Li, W. Journal of Experimental Biology. 2013 Jul 15;216(Pt
14):2702-12. doi: 10.1242/jeb.085746. (1 cit.)

{\em Sequencing of the sea lamprey (Petromyzon marinus) genome provides insights into vertebrate evolution.} Smith JJ et al. Nature Genetics, Apr;45(4):415-21, 421e1-2. doi: 10.1038/ng.2568. Epub 2013 Feb 24. (26 cit.)

{\em Draft Genome Sequences of two Campylobacter jejuni Clinical Isolates,
NW and D2600.} Jerome JP, Klahn BD, Bell JA, Barrick JE, {\bf Brown CT}, Mansfield LS. Journal of Bacteriology, 194 (20), 5707-5708. (2 cit.)

{\em Standing Genetic Variation in Contingency Loci Drives the Rapid
  Adaptation of Campylobacter jejuni to a Novel Host} Jerome JP, Bell
JA, Plovanich-Jones AE, Barrick JE, {\bf Brown CT}, Manfield LS.  PLoS One 6
(1), e16399, Jan 24 2011. (20 cit.)

\vspace{2cm}
{\centerline {Page 3/7}}

\newpage

{\em Exploring the future of bioinformatics data sharing and mining
  with Pygr and Worldbase} Lee C, Alekseyenko A, {\bf Brown CT}.  in {\em
  Proceedings of the 8th Python in Science conference (SciPy 2009)}, G
Varoquaux, S van der Walt, J Millman (Eds.), pp. 62-67.  (4 cit.)

{\em Diverse syntrophic partnerships from deep-sea methane vents revealed by direct cell capture and metagenomics.}\\
Pernthaler A, Dekas AE, {\bf Brown CT}, Goffredi SK, Embaye T, Orphan VJ.\\
Proc Natl Acad Sci U S A. 2008 May
13;105(19):7052-7. Epub 2008 May 8.  PMID: 18467493.  (139 cit.)

{\em The genome of the sea urchin Strongylocentrotus purpuratus.}\\
Sea Urchin Genome Sequencing Consortium.\\
Science. 2006 Nov 10;314(5801):941-52.
PMID: 17095691.  (432 cit.)

{\em High regulatory gene use in sea urchin embryogenesis: Implications for bilaterian development and evolution.}\\
Howard-Ashby M, Materna SC, {\bf Brown CT}, Tu Q, Oliveri P, Cameron RA, Davidson EH.\\
Dev Biol. 2006 Dec 1;300(1):27-34. Epub 2006 Oct 18.
PMID: 17101125.  (30 cit.)

{\em Gene families encoding transcription factors expressed in early development of Strongylocentrotus purpuratus.}
Howard-Ashby M, Materna SC, {\bf Brown CT}, Chen L, Cameron RA, Davidson EH.\\
Dev Bio 2006 300 (1), 90-107 (83 cit.)

{\em Identification and characterization of homeobox transcription factor genes in Strongylocentrotus purpuratus, and their expression in embryonic development}
Howard-Ashby M, Materna C, {\bf Brown CT}, Chen L, Cameron RA, Davidson EH.
2006 Dev Bio 300 (1), 74-89.  (65 cit.)

{\em Sea urchin Forkhead gene family: phylogeny and embryonic expression}
Tu Q, {\bf Brown CT}, Davidson EH, Oliveri P.
2006 Dev Bio 300 (1), 49-62. (86 cit.)

{\em Paircomp, FamilyRelationsII and Cartwheel: tools for interspecific sequence comparison.}\\
{\bf Brown CT}, Xie Y, Davidson EH, Cameron RA.\\
BMC Bioinformatics. 2005 Mar 24;6:70.
PMID: 15790396 (28 cit.)

{\em Anaerobic regulation by an
atypical Arc system in {\em Shewanella oneidensis}.}\\
Gralnick JA, {\bf Brown CT}, Newman DK.\\
Mol Microbiol. 2005
Jun;56(5):1347-57.  PMID: 15882425 (48 cit.)

{\color{red} ${\bf \triangleright}$}
{\em Evolutionary comparisons suggest
many novel cAMP response protein binding sites in {\em E. coli}.}\\
{\bf Brown CT}, Callan CG Jr.\\
Proc Natl Acad Sci U S A. 2004 Feb 24;101(8):2404-9.  PMID: 14983022 (46 cit.)

{\em Patchy interspecific sequence similarities efficiently identify positive cis-regulatory elements in the sea urchin.}
Yuh CH, {\bf Brown CT}, Livi CB, Rowen L, Clarke PJC, Davidson EH.
2002 Dev Bio 246 (1), 148-161. (82 cit.)

  {\em New
computational approaches for analysis of {\em cis}-regulatory networks.}  \\
{\bf Brown CT}, Rust AG, Clarke PJ, Pan Z, Schilstra MJ, De Buysscher
T, Griffin G, Wold BJ, Cameron RA, Davidson EH, Bolouri H.\\
Dev Biol. 2002 Jun 1;246(1):86-102.  PMID: 12027436 (104 cit.)

{\em A genomic
regulatory network for development.}\\
Davidson EH, Rast JP, Oliveri P, Ransick A, Calestani C, Yuh CH,
Minokawa T, Amore G, Hinman V, Arenas-Mena C, Otim O, {\bf Brown CT}, Livi
CB, Lee PY, Revilla R, Rust AG, Pan Z, Schilstra MJ, Clarke PJ, Arnone
MI, Rowen L, Cameron RA, McClay DR, Hood L, Bolouri H.
\\Science. 2002 Mar
1;295(5560):1669-78.  PMID: 11872831. (1050 cit.)

\vspace{1cm}
{\centerline {Page 4/7}}

\newpage



{\em A provisional regulatory gene network for specification of endomesoderm in the sea urchin embryo.}
Davidson EH, Rast JP, Oliveri P, Ransick A, Calestani C, Yuh CH, Minokawa T, Amore G, Hinman V, Arenas-Mena C, Otim O, Brown CT, Livi CB, Lee PY, Revilla R, Schilstra MJ, Clarke PJ, Rust AG, Pan Z, Arnone MI, Rowen L, Cameron RA, McClay DR, Hood L, Bolouri H.
Dev Biol. 2002 Jun 1;246(1):162-90.
PMID: 12027441 (227 cit.)

{\em The Earthshine Project: update on photometric and spectroscopic measurements.}
 E. Palle, P. M. Rodriguez, P. R. Goode, J. Qiu, V. Yurchyshyn, J. Hickey, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin.
Solar Variability and Climate Change Advances in Space Research 34, 288 (2004).

{\em The earthshine spectrum}
P. M. Rodriguez, E. Palle, P. R. Goode, J. Hickey, J. Qiu, V. Yurchyshyn, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin.
Solar Variability and Climate Change Advances in Space Research 34, 293 (2004).

{\em Sunshine, Earthshine and Climate Change: II. Solar Origins of Variations in the Earth's Albedo.}  P.R. Goode, E. Palli,V. Yurchyshyn, J. Qiu,
 J. Hickey, P. Montaqis-Rodriguez,M.-C. Chu,
 E. Kolbe,C.T. Brown, S.E. Koonin.
Journal of the Korean Astronomical Society, 35, 1 (2003).

{\em Earthshine and the Earth's albedo: 1. Earthshine observations and measurements of the lunar phase function for accurate measurements of the Earth's Bond albedo}
J. Qiu, P. R. Goode, E. Palle, V. Yurchyshyn, J. Hickey, P. M. Rodriguez, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin.
J. of Geophys. Res.-Atmospheres 108, 4709 (2003).  (33 cit.)

{\em Earthshine and the Earth's albedo: 2. Observations and simulations over three years}
 E. Palle, P. R. Goode, V. Yurchyshyn, J. Qiu, J. Hickey, P. M. Rodriguez, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin.
J. of Geophys. Res.-Atmospheres 108, 4710 (2003). (43 cit.)


{\em Earthshine observations of the earth's reflectance}
P. R. Goode, J. Qiu, V. Yurchyshyn, J. Hickey, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin
Geophys. Res. Lett. 28, 1671 (2001). (66 cit.)

{\em Visualizing Evolutionary Activity of Genotypes in Evita};
with M. Bedau.  Adaptive Systems, 1998. (51 cit.)

{\em A Comparison of Evolutionary Activity in Artificial Living Systems and in the 
	Biosphere;} Snyder E, Brown CT, Bedau M, Packard N.
in the Proceedings of the 4th Europ. Conf. on 
Artificial Life, July, 1997.  (60 cit.)

{\em Abundance Distributions in Artificial Life and Stochastic Models: "Age and
Area" revisited}, Adami, C., Brown, C.T., Haggerty, M.R.
Proc. of 3rd Europ. Conf. on Artificial Life, June 4-6, 1995,
Granada, Spain, Lecture Notes in Computer Science, Springer Verlag (1995),
p.503. (16 cit.)

{\em Evolutionary Learning in the 2D Artificial Life System ``Avida''}\\
Adami C, Brown CT. Proc. of ``Artificial Life IV'', MIT Press, p. 377-381
(1994).  (173 cit.)

\section{\mysidestyle Invited Articles and Reviews}

{\em Strain recovery from metagenomes. Brown CT.} Nature Biotechnology
33 (10), 1041-1043, Oct 2015.

{\em Metagenomics: the paths forward}.  Brown CT and Tiedje JM.
Handbook of Molecular Microbiology II: Metagenomics in Different
Habitats.  Wiley-Blackwell 10 Nov 2011.

{\em Computational approaches to finding and analyzing cis-regulatory elements.} 
Brown CT. Methods Cell Biol. 2008;87:337-65. Review.
PMID: 18485306  (7 cit.)

{\em Reproducible Bioinformatics Research for Biologists}.  Preeyanon
L, Pyrkosz AB, and Brown CT. Chapter in Implementing Reproducible
Computational Research, V. Stodden and R. Peng, ed.  Forthcoming in
Dec 2013.

\vspace{1cm}
{\centerline {Page 5/7}}

\newpage

\section{\mysidestyle Reports and Editorials}

{\em Cephalopod genomics: A plan of strategies and organization.} Albertin et al., Standards in Genomic Sciences 7 (1), 175. (2 cit.)

{\em Changing computational research. The challenges ahead.}
Neylon, C., Aerts, J., Brown, C.T., Lemire, D., Millman, J., Murray-Rust, P., Perez, F., Saunders, N., Smith, A., Varoquaux, G.
Source Code for Biology and Medicine 7 (1), 2 (2012). (3 cit.)

{\em Meeting report: the terabase metagenomics workshop and the vision of an Earth microbiome project.}
Gilbert, J.A., Meyer, F., Antonopoulos, D., Balaji, P., Brown, C.T., Brown, C.T., Desai, N., Eisen, J.A., Evers, D., Field, D. Standards in genomic sciences, 3(3) 243 (2010). (41 cit.)

\section{\mysidestyle Online commentaries, blogs, and social media}

Personal/professional blog at: \href{http://ivory.idyll.org/blog/}{ivory.idyll.org/blog/}.  A few selected posts (click on links): \href{http://ivory.idyll.org/blog/replication-i.html}{``Our approach to replication in computational science''}, \href{http://ivory.idyll.org/blog/thoughts-on-assemblathon-2.html}{``Thoughts on Assemblathon 2''}, \href{http://ivory.idyll.org/blog/the-future-of-khmer-2013-version.html}{``The future of khmer (2013)''}.

Twitter: \href{http://twitter.com/ctitusbrown}{@ctitusbrown}

\href{http://blogs.biomedcentral.com/bmcblog/2013/02/28/version-control-for-scientific-research/}{BioMedCentral invited blog post: ``Version control for scientific research''}

\section{\mysidestyle Selected Invitations and Meetings}

\begin{list1}

\item[] April 2013 - NIH NHGRI Education and Training committee.
\item[] March 2013 - Invited speaker at National Center for Atmospheric Research Software Engineering Assembly.
\item[] March 2013 - NSF/Moore Foundation meeting on Cyberinfrastructure for Marine 'Omics.
\item[] September 2012 - Invited speaker at Extremely Large Databases 2012 (XLDB 2012).
\item[] June 2012 - NSF BIO Centers meeting on Cyberinfrastructure Needs in BIO.

\end{list1}

    %____________________________________________________________________________________
    \section{\mysidestyle Professional\\Activities}

\begin{list1}
\item[] iPlant Scientific Advisory Board member.
\item[] Software Carpentry Scientific Advisory Board member.
\item[] Accredited Software Carpentry Instructor.
\item[] NIH Committee Member for Cloud Computing and the Human Microbiome.
\item[] Cephalopod Genome Sequencing Consortium Steering Committee, 2012-present.
\item[] Member of the Editorial Board for Open Research Computation,
Frontiers in Livestock Genomics.
\item[] Xconomist.com, invited member, Advisory Board (Michigan chapter).
\item[] BEACON NSF STC, Thrust Group co-leader (responsible for reviewing
proposals, organizing activities), 2010-2014.
\item[] Course director, 2010-present, Next-Generation Sequence Analysis for Biologists, KBS, MSU.
\item[] Course faculty, 2006-2008, Embryology Course, Woods Hole Marine Biological Laboratory.
\item[] Founder, Caltech Bioinformatics Journal Club; biology-in-python
mailing list.
\item[] Faculty advisor, Metagenomics Journal Club at MSU.
\item[] Development and maintenance of several open source bioinformatics tools, including
Cartwheel server for comparative sequence analysis, khmer k-mer software,
and screed; github.com/ctb/.
\item[] Active in open source testing community: twill, figleaf, pony-build.
\item[] Reviewer for National Science Foundation; Developmental Biology, BMC Bioinformatics, BMC Genomics, Genome Biology, Bioinformatics, PLoS One.
\end{list1}

\newpage

\section{\mysidestyle Teaching and Workshops}

\begin{list1}

\item[] Open Problems in Bioinformatics, CSE/MMG graduate seminar
  course (2008-2009)
\item[] Database-Backed Web Development, CSE 4xx (2008-, yearly)
\item[] Introduction to Computational Science for Evolutionary Biologists, CSE 801 (was 891) (2010-, yearly).
\item[] Analyzing Next-Generation Sequencing Data, research workshop (2010-, yearly).
\item[] Software Carpentry workshops: Scripps Research Institute (11/2012), U. Arizona (4/2013)
\item[] Instructor at Marine Biological Laboratory course on Strategies and Techniques for Analyzing Microbial Population Structures, 2012 and 2013.
\item[] Co-instructor for Workshop on Microbial Bioinformatics, 10/2013, Caltech.
\item[] Lead instructor for Workshop on mRNAseq for Biologists, and Workshop for Advanced Bioinformatics Developers, 11/2013, The Centre For Genome Analysis, Norwich, UK.
\end{list1}

\section{\mysidestyle Former Students}

Jiarong Guo (MS in Fisheries and Wildlife, 2010).  Thesis topic:
Phylogenetic analysis of annotations for uncultured bacteria.
Currently working on his PhD in Microbiology at MSU with Dr. James
Tiedje and myself.

Dr. Jason Pell (PhD in Computer Science, MSU, 2013). Thesis topic:
Efficient algorithms for the analysis of sequencing data.  Currently
working at Google.

Dr. Likit Preeyanon (PhD in Microbiology and Molecular Genetics, MSU,
2014). Thesis topic: Exploring mechanisms of genetic resistance to
Marek's Disease.  Professor (permanent position) at Mahidol
University, Thailand (2014-present).

Dr. Qingpeng Zhang (PhD in Computer Science, MSU, 2015).  Thesis
topic: A novel strategy for analyzing metagenomic sequence from many
samples.  Postdoc at Joint Genome Institute (2015).

Dr. Elijah Lowe (PhD in Computer Science, MSU, 2015).  Thesis topic:
Evolutionary developmental biology and genomics of the Molgulid
ascidians.  Postdoc at Stazione Zoologica in Naples (2014-present).

\section{\mysidestyle Current Students}

\begin{list1}
\item[] Jiarong Guo (MMG PhD student at MSU, 2010-2015 (expected))
\item[] Camille Scott (CSE PhD student, 2012-2017 (expected))
\item[] Luiz Irber (CSE PhD student, 2014-2019 (expected))
\item[] Lisa Cohen (Physiology PhD student, 2015-2020 (expected)).
\end{list1}

%________________________________________________________________________________________

\section{\mysidestyle Postdoctoral trainees}

\begin{list1}
\item[] Dr. Adina Chuang Howe (now Assistant Professor at Iowa State)
\item[] Dr. Kanchan Pavangadkar (now undergraduate advisor at MSU)
\item[] Dr. Sherine Awad (current postdoctoral fellow)
\item[] Dr. Tamer Mansour (current postdoctoral fellow)
\end{list1}

\section{\mysidestyle References}

{\em (Contact details available upon request.)}

Dr. Ewan Birney, Associate Director of the EMBL-EBI.

Professor Jonathan Eisen, University of California Davis.

Professor Paul W. Sternberg, California Institute of Technology.

Professor Billie J. Swalla, University of Washington at Seattle.

\vspace{2cm}
{\centerline {Page 7/7}}

\end{resume}

\end{document}

%________________________________________________________________________________________
% EOF


