
%________________________________________________________________________________________
% @brief    LaTeX2e Resume for C. Titus Brown
% @author   Titus Brown
% @date     January 2008
% @info     Based on Latex Resume Template by Chris Paciorek 
%           http://www.biostat.harvard.edu/~paciorek/


%________________________________________________________________________________________
\documentclass[margin,line]{resume}
\begin{document}
\name{\Large C. Titus Brown}
\pagenumbering{arabic}
\begin{resume}


    %____________________________________________________________________________________

    % Education
    \section{\mysidestyle Education}

    {\bf Reed College}, Portland, OR; Mathematics; B.A., 1997

    \vspace{2mm}

    {\bf California Institute of Technology}, Davidson Lab (graduate student);
\\
Developmental Biology; PhD., 2007

    \vspace{2mm}

    {\bf California Institute of Technology}, Bronner-Fraser Lab (postdoc);\\
 Developmental Biology and Bioinformatics; 2007-2008

    \vspace{2mm}

    \section{\mysidestyle Appointments}

    {\bf Assistant Professor}, 
Microbiology \& Molecular Genetics / Computer Science and Engineering\\
Michigan State University, 2008-present.

    %____________________________________________________________________________________
    % Honours and Awards
    \section{\mysidestyle Honours and\\Awards} 

Burroughs-Wellcome Fund Computational Biology Fellowship (1999-2004).\\
Withrow Award for Teaching Excellence (2008-2009).

    \section{\mysidestyle Grants}

1. USDA, \$690,000; 12/2009-11/2013 (100\%).  PI, ``Easily accessible Web-based tools for analyzing next-generation sequencing data from agricultural animals''.  

2. NSF, \$50,000; 9/2009-8/2011 (100\%).  PI, ``RV1: MSB: Collaborative: Symbiont Separation and Investigation of the Novel Heterotrophic Osedax Symbiosis using Comparative Genomics''.

3. USDA, \$99,000; 2/2009-12/2011 (100\%).  PI, ``Positional Candidate Genes for Resistance to Marek's
Disease by Screening for Marek's Disease Virus Meq-regulated Genes''.

4. NIH (R25 education), \$104,000 (100\%); 7/2011-6/2014.  PI, ``Analyzing Next-Generation Sequencing Data''.

5. DOE, \$659,587 (5\%).  9/2011-8/2012.  co-PI, ``Ribosomal Database Project''.

6. USDA (Grad Fellowships), \$238,000 (15\%).  1/2012-11/2017.  co-PI, ``Integrated genomics training program.''

7. NSF OCI Supplement to BEACON STC, \$200k (100\%); 1/1/2013-12/31/2013.  PI, ``Materials and Workshops for Cyberinfrastructure Education in Biology.''

8. USDA, \$2,989,032 (5.8\%). 1/2013-1/2017.  co-PI, ``The Genetics of Johne's Disease.''

    %____________________________________________________________________________________
    % Publications
    \section{\mysidestyle Submitted Manuscripts}

{\em Assembling large, complex environmental metagenomes.} Howe AC, Jansson J, Malfatti SA, Tringe SG, Tiedje JM, {\bf Brown CT}. preprint arXiv:1212.2832.

{\em Illumina Sequencing Artifacts Revealed by Connectivity Analysis of Metagenomic Datasets.} Howe AC, Pell J, Canino-Koning R, Mackelprang R, Tringe SG, Jansson J, Tiedje JM, {\bf Brown CT}. preprint arXiv:1212.0159.

{\em Best practices for scientific computing.} Wilson GV et al. preprint arXiv:1210.0530.

{\em A Reference-Free Algorithm for Computational Normalization of Shotgun Sequencing Data.} {\bf Brown CT}, Howe AC, Zhang Q, Pyrkosz AB, Brom TH. preprint arXiv:1203.4802. (2 cit.)

\newpage

    \section{\mysidestyle Peer Reviewed Publications}
{\em Full publication list at: http://scholar.google.com/citations?user=O4rYanMAAAAJ\\
April 2013: 3001 citations total; h-index of 20, i10-index of 23.}

{\em Sequencing of the sea lamprey (Petromyzon marinus) genome provides insights into vertebrate evolution.} Smith JJ et al. Nature Genetics, published online Feb 2, 2013. (1 cit.)

{\em Draft Genome Sequences of two Campylobacter jejuni Clinical Isolates,
NW and D2600.} Jerome JP, Klahn BD, Bell JA, Barrick JE, {\bf Brown CT}, Mansfield LS. Journal of Bacteriology, 194 (20), 5707-5708. (1 cit.)

{\em Scaling metagenome sequence assembly with probabilistic de Bruijn
graphs.} Pell J, Hintze A, Canino-Koning R, Howe A, Tiedje JM, {\bf Brown
  CT}. Proc Natl Acad Sci USA, published online before print July 30,
  2012, doi: 10.1073/pnas.1121464109. (11 cit.)

{\em Standing Genetic Variation in Contingency Loci Drives the Rapid
  Adaptation of Campylobacter jejuni to a Novel Host} Jerome JP, Bell
JA, Plovanich-Jones AE, Barrick JE, {\bf Brown CT}, Manfield LS.  PLoS One 6
(1), e16399, Jan 24 2011. (12 cit.)

{\em Exploring the future of bioinformatics data sharing and mining
  with Pygr and Worldbase} Lee C, Alekseyenko A, {\bf Brown CT}.  in {\em
  Proceedings of the 8th Python in Science conference (SciPy 2009)}, G
Varoquaux, S van der Walt, J Millman (Eds.), pp. 62-67.  (2 cit.)

{\em Diverse syntrophic partnerships from deep-sea methane vents revealed by direct cell capture and metagenomics.}\\
Pernthaler A, Dekas AE, {\bf Brown CT}, Goffredi SK, Embaye T, Orphan VJ.\\
Proc Natl Acad Sci U S A. 2008 May
13;105(19):7052-7. Epub 2008 May 8.  PMID: 18467493.  (110 cit.)

{\em The genome of the sea urchin Strongylocentrotus purpuratus.}\\
Sea Urchin Genome Sequencing Consortium.\\
Science. 2006 Nov 10;314(5801):941-52.
PMID: 17095691.  (432 cit.)

{\em High regulatory gene use in sea urchin embryogenesis: Implications for bilaterian development and evolution.}\\
Howard-Ashby M, Materna SC, {\bf Brown CT}, Tu Q, Oliveri P, Cameron RA, Davidson EH.\\
Dev Biol. 2006 Dec 1;300(1):27-34. Epub 2006 Oct 18.
PMID: 17101125.  (28 cit.)

{\em Gene families encoding transcription factors expressed in early development of Strongylocentrotus purpuratus.}
Howard-Ashby M, Materna SC, {\bf Brown CT}, Chen L, Cameron RA, Davidson EH.\\
Dev Bio 2006 300 (1), 90-107 (74 cit.)

{\em Identification and characterization of homeobox transcription factor genes in Strongylocentrotus purpuratus, and their expression in embryonic development}
Howard-Ashby M, Materna C, {\bf Brown CT}, Chen L, Cameron RA, Davidson EH.
2006 Dev Bio 300 (1), 74-89.  (60 cit.)

{\em Sea urchin Forkhead gene family: phylogeny and embryonic expression}
Tu Q, {\bf Brown CT}, Davidson EH, Oliveri P.
2006 Dev Bio 300 (1), 49-62. (69 cit.)

{\em Paircomp, FamilyRelationsII and Cartwheel: tools for interspecific sequence comparison.}\\
{\bf Brown CT}, Xie Y, Davidson EH, Cameron RA.\\
BMC Bioinformatics. 2005 Mar 24;6:70.
PMID: 15790396 (21 cit.)

{\em Anaerobic regulation by an
atypical Arc system in {\em Shewanella oneidensis}.}\\
Gralnick JA, {\bf Brown CT}, Newman DK.\\
Mol Microbiol. 2005
Jun;56(5):1347-57.  PMID: 15882425 (38 cit.)

\newpage

{\em Evolutionary comparisons suggest
many novel cAMP response protein binding sites in {\em E. coli}.}\\
{\bf Brown CT}, Callan CG Jr.\\
Proc Natl Acad Sci U S A. 2004 Feb 24;101(8):2404-9.  PMID: 14983022 (43 cit.)

{\em Patchy interspecific sequence similarities efficiently identify positive cis-regulatory elements in the sea urchin.}
Yuh CH, {\bf Brown CT}, Livi CB, Rowen L, Clarke PJC, Davidson EH.
2002 Dev Bio 246 (1), 148-161. (79 cit.)

  {\em New
computational approaches for analysis of {\em cis}-regulatory networks.}  \\
{\bf Brown CT}, Rust AG, Clarke PJ, Pan Z, Schilstra MJ, De Buysscher
T, Griffin G, Wold BJ, Cameron RA, Davidson EH, Bolouri H.\\
Dev Biol. 2002 Jun 1;246(1):86-102.  PMID: 12027436 (92 cit.)

{\em A genomic
regulatory network for development.}\\
Davidson EH, Rast JP, Oliveri P, Ransick A, Calestani C, Yuh CH,
Minokawa T, Amore G, Hinman V, Arenas-Mena C, Otim O, {\bf Brown CT}, Livi
CB, Lee PY, Revilla R, Rust AG, Pan Z, Schilstra MJ, Clarke PJ, Arnone
MI, Rowen L, Cameron RA, McClay DR, Hood L, Bolouri H.
\\Science. 2002 Mar
1;295(5560):1669-78.  PMID: 11872831. (987 cit.)

{\em A provisional regulatory gene network for specification of endomesoderm in the sea urchin embryo.}
Davidson EH, Rast JP, Oliveri P, Ransick A, Calestani C, Yuh CH, Minokawa T, Amore G, Hinman V, Arenas-Mena C, Otim O, Brown CT, Livi CB, Lee PY, Revilla R, Schilstra MJ, Clarke PJ, Rust AG, Pan Z, Arnone MI, Rowen L, Cameron RA, McClay DR, Hood L, Bolouri H.
Dev Biol. 2002 Jun 1;246(1):162-90.
PMID: 12027441 (213 cit.)

{\em The Earthshine Project: update on photometric and spectroscopic measurements.}
 E. Palle, P. M. Rodriguez, P. R. Goode, J. Qiu, V. Yurchyshyn, J. Hickey, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin.
Solar Variability and Climate Change Advances in Space Research 34, 288 (2004).

{\em The earthshine spectrum}
P. M. Rodriguez, E. Palle, P. R. Goode, J. Hickey, J. Qiu, V. Yurchyshyn, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin.
Solar Variability and Climate Change Advances in Space Research 34, 293 (2004).

{\em Sunshine, Earthshine and Climate Change: II. Solar Origins of Variations in the Earth's Albedo.}  P.R. Goode, E. Palli,V. Yurchyshyn, J. Qiu,
 J. Hickey, P. Montaqis-Rodriguez,M.-C. Chu,
 E. Kolbe,C.T. Brown, S.E. Koonin.
Journal of the Korean Astronomical Society, 35, 1 (2003).

{\em Earthshine and the Earth's albedo: 1. Earthshine observations and measurements of the lunar phase function for accurate measurements of the Earth's Bond albedo}
J. Qiu, P. R. Goode, E. Palle, V. Yurchyshyn, J. Hickey, P. M. Rodriguez, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin.
J. of Geophys. Res.-Atmospheres 108, 4709 (2003).  (33 cit.)

{\em Earthshine and the Earth's albedo: 2. Observations and simulations over three years}
 E. Palle, P. R. Goode, V. Yurchyshyn, J. Qiu, J. Hickey, P. M. Rodriguez, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin.
J. of Geophys. Res.-Atmospheres 108, 4710 (2003). (43 cit.)

{\em Earthshine observations of the earth's reflectance}
P. R. Goode, J. Qiu, V. Yurchyshyn, J. Hickey, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin
Geophys. Res. Lett. 28, 1671 (2001). (66 cit.)

{\em Visualizing Evolutionary Activity of Genotypes in Evita};
with M. Bedau.  Adaptive Systems, 1998. (51 cit.)

{\em A Comparison of Evolutionary Activity in Artificial Living Systems and in the 
	Biosphere;} Snyder E, Brown CT, Bedau M, Packard N.
in the Proceedings of the 4th Europ. Conf. on 
Artificial Life, July, 1997.  (60 cit.)

\newpage

{\em Abundance Distributions in Artificial Life and Stochastic Models: "Age and
Area" revisited}, Adami, C., Brown, C.T., Haggerty, M.R.
Proc. of 3rd Europ. Conf. on Artificial Life, June 4-6, 1995,
Granada, Spain, Lecture Notes in Computer Science, Springer Verlag (1995),
p.503. (16 cit.)

{\em Evolutionary Learning in the 2D Artificial Life System ``Avida''}\\
Adami C, Brown CT. Proc. of ``Artificial Life IV'', MIT Press, p. 377-381
(1994).  (173 cit.)

\section{\mysidestyle Invited Articles and Reviews}

{\em Metagenomics: the paths forward}.  Brown CT and Tiedje JM.
Handbook of Molecular Microbiology II: Metagenomics in Different
Habitats.  Wiley-Blackwell 10 Nov 2011.

{\em Computational approaches to finding and analyzing cis-regulatory elements.} 
Brown CT. Methods Cell Biol. 2008;87:337-65. Review.
PMID: 18485306  (5 cit.)

\section{\mysidestyle Reports and Editorials}

{\em Cephalopod genomics: A plan of strategies and organization.} Albertin et al., Standards in Genomic Sciences 7 (1), 175.

{\em Changing computational research. The challenges ahead.}
Neylon, C., Aerts, J., Brown, C.T., Lemire, D., Millman, J., Murray-Rust, P., Perez, F., Saunders, N., Smith, A., Varoquaux, G.
Source Code for Biology and Medicine 7 (1), 2 (2012).

{\em Meeting report: the terabase metagenomics workshop and the vision of an Earth microbiome project.}
Gilbert, J.A., Meyer, F., Antonopoulos, D., Balaji, P., Brown, C.T., Brown, C.T., Desai, N., Eisen, J.A., Evers, D., Field, D. Standards in genomic sciences, 3(3) 243 (2010). (29 cit.)

\section{\mysidestyle Selected Invitations and Meetings}

\begin{list1}

\item[] April 2013 - NIH NHGRI Education and Training committee.
\item[] March 2013 - Invited speaker at National Center for Atmospheric Research Software Engineering Assembly.
\item[] March 2013 - NSF/Moore Foundation meeting on Cyberinfrastructure for Marine 'Omics 
\item[] September 2012 - Invited speaker at Extremely Large Databases 2012
\item[] June 2012 - NSF BIO Centers meeting on Cyberinfrastructure Needs in BIO

\end{list1}

    %____________________________________________________________________________________
    \section{\mysidestyle Professional\\Activities}

\begin{list1}
\item[] NIH Committee Member for Cloud Computing and the Human Microbiome.
\item[] Cephalopod Genome Sequencing Consortium Steering Committee, 2012-present.
\item[] Member of the Editorial Board for Open Research Computation,
Frontiers in Livestock Genomics.
\item[] Xconomist.com, invited member, Advisory Board (Michigan chapter).
\item[] BEACON NSF STC, Thrust Group co-leader (responsible for reviewing
proposals, organizing activities), 2010-.
\item[] Course director, 2010-present, Next-Generation Sequence Analysis for Biologists, KBS, MSU.
\item[] Course faculty, 2006-2008, Embryology Course, Woods Hole Marine Biological Laboratory.
\item[] Founder, Caltech Bioinformatics Journal Club; biology-in-python
mailing list.
\item[] Faculty advisor, Metagenomics Journal Club at MSU.
\item[] Development and maintenance of several open source bioinformatics tools, including
Cartwheel server for comparative sequence analysis, khmer k-mer software,
and screed; github.com/ctb/.
\item[] Active in open source testing community: twill, figleaf, pony-build.
\item[] Reviewer for National Science Foundation; Developmental Biology, BMC Bioinformatics, BMC Genomics, Genome Biology, Bioinformatics, PLoS One.
\end{list1}

\section{\mysidestyle Former Students}

Jiarong Guo (MS in Fisheries and Wildlife, 2010).  Thesis topic:
Phylogenetic analysis of annotations for uncultured bacteria.

\section{\mysidestyle Current Students}

\begin{list1}
\item[] Jiarong Guo (MMG PhD student, 2010-2015 (expected))
\item[] Elijah Lowe (CSE PhD student, 2008-2013 (expected))
\item[] Jason Pell (CSE PhD student, 2009-2014 (expected))
\item[] Likit Preeyanon (MMG PhD student, 2008-2013 (expected)
\item[] Qingpeng Zhang (CSE PhD student, 2008-2013 (expected))
\item[] Chris Welcher (CSE MS student, 2012-2014 (expected))
\end{list1}

%________________________________________________________________________________________
\end{resume}
\end{document}

%________________________________________________________________________________________
% EOF

