## Overview

My pre-faculty research was in developmental molecular biology,
gene networks, and systems biology, but since then I've focused on how
to make (biological) sense of large amounts of sequencing data,
typically from non-model systems (so largely transcriptomes and
environmental metagenomes).  I'm particularly interested in questions
of scaling and data integration, and how to make use of data to move
more quickly to biologically relevant hypotheses.  I have an abiding
fascination with marine evo-devo and soil/sediment metagenomics.  I've
just moved into the School of Veterinary Medicine at UC Davis and
would be very interested in discussing challenges of moving into a
more applied/clinical setting, as well.

On the computational side, I've invested heavily in streaming and
probabilistic data structures as applied to short-read analysis, and
we are starting to work on read-to-graph sequence alignment and genome
reference graphs.

I teach quite a few workshops on next-gen sequence analysis and better
computational training, too.

Finally, I'm pro-open science, open source, and open access, and I use
social media a lot.  Sometimes people are interested in talking about
that.

## Background

* PhD Caltech 2007, Biology; worked on early sea urchin development and genomics with Eric Davidson.
* postdoc Caltech 2007-2008; worked with Marianne Bronner on chick neural crest development;
* Asst Prof, Comp Sci / Microbiology, Michigan State University, 2008-2014; efficient algorithms for sequence analysis
* Assoc Prof, School of VetMed, UC Davis, 2015-present; data intensive biology and genomics.
